.env.dist

```diff
@@ -9,9 +9,6 @@ GOOGLE_CLIENT_SECRET=
 ### yelp/api ###
 YELP_API_KEY=
 
-###> symfony/framework-bundle ###
+### symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=fafa69ad10e0a0e5ee2b6b9ea33afd24
-#TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-#TRUSTED_HOSTS='^localhost|example\.com$'
-###< symfony/framework-bundle ###
```

Comment:
You should leave this line as it is used by Symfony to know which lines to remove if framework-bundle is uninstalled. Also for the > removed before symfony/framework-bundle and in other files, this is used as a marker by symfony ;) not a fancy thing :D